smokey/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -128,8 +134,8 @@ public Command getScoreRoutine(Pose2d NodePosition){
         if(this.subsystems.getPickerSubsystem() != null) {
             command.addCommands(new ManipulatePickerCommand(subsystems.getPickerSubsystem(), true));
         }
-        else if (this.subsystems.getEveryBotPickerSubsystem() != null) {
-            command.addCommands(new EveryBotPickerAutoExpellCommand(subsystems.getEveryBotPickerSubsystem()));
+        else if (this.subsystems.getEveryBotPickerSubsystem() != null) {// cube uses uptake command to expell
+            command.addCommands(new EveryBotPickerAutoUptakeCommand(subsystems.getEveryBotPickerSubsystem()));
```

Comment:
Maybe this command needs a new name.  Maybe UptakeConeExpelCube?
Or we could make one command that is EveryBotPickerCommand that takes an Enum ExpelCube, ExpelCone, UptakeCone, UptakeCube?

Reply:
I think we should decide the UX here so we can accommodate how we think we can best get muscle memory for co-pilot